src/LineageTree/lineageTree.py

```diff
@@ -25,10 +25,13 @@
 import matplotlib.pyplot as plt
 import networkx as nx
 import numpy as np
+from scipy.interpolate import InterpolatedUnivariateSpline
 
 # from edist.uted import uted
+
 from scipy.spatial import Delaunay
 from scipy.spatial import cKDTree as KDTree
+from sklearn.metrics.pairwise import euclidean_distances
```

Comment:
Is there another function to compute the distance matrix? As of now, `sklearn` is not necessary to make lineage tree work and I'd like to keep it that way if possible.

If there is an equivalent function in `numpy` or `scipy` please use it.

Reply:
yes, we can use eucledian from scipy